parachain/runtime/interlay/src/lib.rs

```diff
@@ -94,7 +94,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     authoring_version: 1,
     spec_version: 1,
     impl_version: 1,
-    transaction_version: 1,
+    transaction_version: 3,
```

Comment:
Why `3`?

Reply:
So that it's the same for all the runtime. It doesn't matter a ton, I can revert if you want

Reply:
We should really increment separately, not least because we will now check these versions in the client and we may use two runtimes with the same name.